### PR TITLE
From lazy to busy

### PR DESCRIPTION
**Pull request by why-not-try-calmer**
This document offers a simple and fresh perspective on how to approach safety on Tumbleweed.

It also suggests a slightly different structure for the table of contents.

OK ~~Note to self: rebase on `dev` after merging #53.~~

### CODE REVIEW

**Comment by why-not-try-calmer** [comment]
@apinter any comments?

File: project/docs/safety_usability.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Doing so means that users will be more often using packages from the official repositories and get familiar with the typical openSUSE tools (such as _Zypper_, _Snapper_ and _Yast_) and configuration scenarios. In the long run this will strengthen their understanding of Linux and of the openSUSE distributions, while improving their ability to troubleshoot with autonomy. 
+
+At scale the openSUSE Project benefits more from users in position to report bugs, provide feedback and make direct contributions.
+
+### openSUSE MicroOS as desktop workstation
+
+Although it is not recommended for new users, it is possible to use MicroOS on a desktop computer, taking advantage of transactional updates. Consider our [guide to get started](/microos_getting_started).
+
+This might appeal to advanced users who agree to spend more time configuring their system in exchange for a system that will require little to no maintenance (especially suited for friends and relatives with little to no experience of Linux or openSUSE).
```
Comment (apinter) [inline project/docs/safety_usability.md:48]: I think the whole point of MicroOS is to not configure as much as one would on a regular desktop like TW. The whole idea of MicroOS is to set it up once and forget about it while using it for one thing only - in this case as a Desktop -. I think this sentence's second half is contradicting the first and overall contradicting the sentence at line 46. 
If it's not recommended for new user why recommend it to "friends and relatives" ^_^ 
Probably need to word it a bit better.

**Review by apinter — CHANGES_REQUESTED** [review]

File: project/docs/safety_usability.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+This is a workflow where the user minimizes the need for manual interventions on their running installation. The way is fine for most users who:
+
+- don't want to spend time configuring their system; or 
+- don't want to take any risk associated with maintenance routines.
+
+Here are some rules of thumb for following this approach:
+
+1. When installing user applications, favour containerized versions (ideally _flatpaks_, see [here](/alternative_procurement#flatpaks)). Use software from the official repositories only when no containerized version is available.
+2. Do not manually install important system libraries or codecs directly from _zypper_ or _Yast2_. Use [opi](/best-of-post#codecs) if you must, but prefer software that ship with their own static libraries and codecs when possible (or flatpaks).
+3. Update every other week or twice a month, not more often. Use `sudo zypper dup`, plus if that applies to you, `flatpak update`.
```
[the remark this answers is not in the document]
Comment (apinter) [reply project/docs/safety_usability.md:32]: FTR the flatpak units are just a suggestion, do ignore if not needed.

File: project/docs/safety_usability.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+## Safety and usability on Tumbleweed
+
+There are two "don't-s" to keep in mind when using Tumbleweed:
+
+- don't use the "1-click installers" available from https://software.opensuse.org
```
[the remark this answers is not in the document]
Comment (why-not-try-calmer) [reply project/docs/safety_usability.md:5]: Okay, but let's not take risks, how long will this linger around, I don't know. I don't want to take bets. Every soul we can save from these traps should be saved.

File: project/docs/safety_usability.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+This is a workflow where the user minimizes the need for manual interventions on their running installation. The way is fine for most users who:
+
+- don't want to spend time configuring their system; or 
+- don't want to take any risk associated with maintenance routines.
+
+Here are some rules of thumb for following this approach:
+
+1. When installing user applications, favour containerized versions (ideally _flatpaks_, see [here](/alternative_procurement#flatpaks)). Use software from the official repositories only when no containerized version is available.
+2. Do not manually install important system libraries or codecs directly from _zypper_ or _Yast2_. Use [opi](/best-of-post#codecs) if you must, but prefer software that ship with their own static libraries and codecs when possible (or flatpaks).
```
[the remark this answers is not in the document]
Comment (why-not-try-calmer) [reply project/docs/safety_usability.md:31]: I 100% agree. Currently the wording of this point implies: flatpaks > self-contained static libs > opi. My reasoning is that someone cannot find or use an application within the first two groups, it's ok for them to use opi.

File: project/docs/safety_usability.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (why-not-try-calmer) [reply project/docs/safety_usability.md:32]: > "not more often" - that doesn't seem right. This bit seem to encourage a use of TW that will get the user stuck on old snapshots and I would recommend removing this. Maybe reword it as: "3. Update at least every other week or twice a month. Use `sudo zypper dup`, plus if that applies to you, `flatpak update`." Flatpaks can be automatically updated with `Gnome-software` or `Discover`. If, however we recommend against using those tools here's a short systemd unit to take care of the updates automatically:
> 
> ```
> # vim /etc/systemd/system/update-flatpaks.timer
> [Unit]
> Description=Update user Flatpaks daily
> 
> [Timer]
> OnCalendar=daily
> Persistent=true
> 
> [Install]
> WantedBy=timers.target
> 
> # vim /etc/systemd/system/update-flatpaks.service
> [Unit]
> Description=Update user Flatpaks
> 
> [Service]
> Type=oneshot
> ExecStart=/usr/bin/flatpak update -y
> 
> [Install]
> WantedBy=default.target
> ```
> 
> Enabling the timer will update all the system-wide installed flatpaks. This can be also modified to do the same thing for user installed applications, but executing `/usr/bin/flatpak --user update -y` instead and move the units under `~/.config/systemd/user/`.

Sold. I love this systemd unit. Do you want to make a quick PR to add it to the `alternative_procurement.md` document? It's really good.

File: project/docs/safety_usability.md (hunk 1 above)
Comment (why-not-try-calmer) [reply project/docs/safety_usability.md:48]: Good points. The intention of this passage I think (I cannot really comment for @syds-git) but from what I remember reviewing his text with him) was to make the following 3 claims:
- setting up MicroOS takes a little bit more time/effort than setting up Leap or TW (I have never used MicroOS, I have no idea, I am just listening at doors!)
- the tradeoff you get from the above is that you need less time/effort to update
- the tradeoff you get from the above makes MicroOS suitable for friends and relatives with little knowledge _provided_ their  machines are set up by an advanced user with the knowledge.

**Comment by why-not-try-calmer** [comment]
Thank you very much @apinter for all these comments. I'll stand-by for your replies and then update the document in a single pass, to avoid back-and-forth.

File: project/docs/safety_usability.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (apinter) [reply project/docs/safety_usability.md:5]: What I mean is that users shouldn't use it at all on any distros till it is available ^_^

File: project/docs/safety_usability.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (apinter) [reply project/docs/safety_usability.md:31]: Ok, gotcha. Yea read it again helped ;)

File: project/docs/safety_usability.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (apinter) [reply project/docs/safety_usability.md:32]: Sure, can do tomorrow morning.

File: project/docs/safety_usability.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (why-not-try-calmer) [reply project/docs/safety_usability.md:31]: If you had to read it twice, this is a bug. Will simplify.

File: project/docs/safety_usability.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (why-not-try-calmer) [reply project/docs/safety_usability.md:5]: Okay, so it's fine, nothing to change there? I mean this document is specifically about TW.

File: project/docs/safety_usability.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (why-not-try-calmer) [reply project/docs/safety_usability.md:32]: <3

File: project/docs/safety_usability.md (hunk 1 above)
Comment (why-not-try-calmer) [reply project/docs/safety_usability.md:48]: Okay, thanks.

File: project/docs/safety_usability.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (apinter) [reply project/docs/safety_usability.md:5]: Yep, fine.

**Comment by why-not-try-calmer** [comment]
Are we mergeable?
